modular_skyrat/modules/implants/code/augments_head.dm

```diff
@@ -0,0 +1,177 @@
+	desc = "Activates your Qani-Laaca computer and grants you its powers. LMB: Short, safer activation. RMB: Longer, more powerful, more dangerous activation."
+	button_icon = 'modular_skyrat/modules/implants/icons/implants.dmi'
+	button_icon_state = "sandy"
+	check_flags = AB_CHECK_CONSCIOUS
+	cooldown_time = 5 MINUTES
+	text_cooldown = TRUE
+	/// Keeps track of how much twitch we inject into people on activation
+	var/injection_amount = 10
+
+/datum/action/cooldown/sensory_enhancer/Trigger(trigger_flags, atom/target)
+	. = ..()
+
+	injection_amount = 10
+
+	if(trigger_flags & TRIGGER_SECONDARY_ACTION)
+		injection_amount = 20
+
+/datum/action/cooldown/sensory_enhancer/Activate(atom/target)
+	. = ..()
+
+	var/mob/living/carbon/human/human_owner = owner
+
+	human_owner.reagents.add_reagent(/datum/reagent/drug/twitch, injection_amount)
+
+	owner.visible_message(span_danger("[owner.name] jolts suddenly as two small glass vials are fired from ports in the implant on their spine, shattering as they land."), \
+			span_userdanger("You jolt suddenly as your Qani-Laaca system ejects two empty glass vials rearward, shattering as they land."))
+	playsound(human_owner, 'sound/items/hypospray.ogg', 50, TRUE)
+
+// Hackerman deck, lets you emag or doorjack things (NO CYBORGS) within a short range of yourself
+
+/obj/item/organ/internal/cyberimp/hackerman_deck
+	name = "Binyat wireless hacking system"
+	desc = "A rare-to-find neural chip that allows its user to interface with nearby machinery \
+		and effect it in (usually) beneficial ways. Due to the rudimentary connection, fine manipulation \
+		isn't possible, however the deck will drop a payload into the target's systems that will attempt \
+		hacking for you. Due to their complexity, the system does not appear to work on cyborgs."
+	icon = 'modular_skyrat/modules/implants/icons/implants.dmi'
+	icon_state = "hackerman"
+	slot = ORGAN_SLOT_BRAIN_ANTISTUN
+	zone = BODY_ZONE_HEAD
+	implant_overlay = null
+	implant_color = null
+	actions_types = list(/datum/action/cooldown/spell/pointed/hackerman_deck)
+	w_class = WEIGHT_CLASS_SMALL
+
+/datum/action/cooldown/spell/pointed/hackerman_deck
+	name = "Activate Ranged Hacking"
+	desc = "Click on any machine, excepting cyborgs, to hack them. Has a short range, only two tiles."
+	active_msg = "You warm up your Binyat deck, there's an idle buzzing at the back of your mind as it awaits a target."
+	deactive_msg = "Your hacking deck makes an almost disappointed sounding buzz at the back of your mind as it powers down."
+	button_icon = 'modular_skyrat/modules/implants/icons/implants.dmi'
+	button_icon_state = "hackerman"
+	spell_requirements = SPELL_REQUIRES_MIND
+	cast_range = 2
+	aim_assist = FALSE
+	spell_max_level = 1 // God I hate actions
+	cooldown_time = 5 MINUTES
+	sparks_amt = 2
+	ranged_mousepointer = 'icons/effects/mouse_pointers/override_machine_target.dmi'
+	/// What we don't work on, will always not work on mobs because I know what you are
```

Comment:
There's still quite a few things that need blacklisting on here. Shuttle console, for example, so people can't send crew to syndieland. Recycler too, considering its only purpose is to gib people.
Honestly, I'm not sure the coolness/benefits of this outweigh the risks of giving pretty much anyone an emag.